web-playground/.gitignore

```diff
@@ -1,24 +1,24 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-/.pnp
-.pnp.js
-
-# testing
-/coverage
-
-# production
-/build
-
-# misc
-.cache
-.DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
-
+# Logs
+logs
+*.log
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+.idea
+.DS_Store
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```

Comment:
I don't like to have editor-specific configuration in the repository if I can help it.

Reply:
This is just the default `.gitignore` that Vite created. I can of course change it, but I actually like letting team members add whatever they like to `.gitignore`, so they can use the tools they want with no risk of cluttering up the repo with files that don't belong there.

I even go one step further, and like to have shared config for various editors too, when a project demands certain config settings (which means I wouldn't actually ignore `.vscode/*`).